src/niscope/metadata/functions_addon.py

```diff
@@ -4,4 +4,146 @@
 functions_override_metadata = {
 }
 
+functions_additional_get_self_cal_last_date_and_time = {
+    'FancyGetLastSelfCalDateAndTime': {
```

Comment:
Shouldn't this be `FancyGetSelfCalLastDateAndTime`?

Applies to other fancy methods too.